personio-personnel-data-api-oa3.yaml

```diff
@@ -3808,11 +3949,11 @@ components:
         half_day_start:
           type: boolean
           example: false
-          description: Weather the start date is a half-day off.
```

Comment:
🌦️ 😄

Reply:
In my defense that was copied blindly from the existing daily period creation, my bad but that was a trap 😹